packages/graphql-hooks/src/middlewares/README.md

```diff
@@ -25,6 +25,22 @@ Middleware is a function with 2 parameters:
   - `resolve`, `reject` - advanced usage only, these would be used only when we want to resolve/reject the request `Promise` early without doing the usual fetch (see [example](examples/cacheMiddleware.js))
 - `next` function - calls the next middleware function in line. Generally it should always be called, unless we want to change the control flow
 
+## Automatic Persisted Queries (APQ) middleware
```

Comment:
okay I'm not sure about this decision. why are we not including this automatically?

Reply:
I thought it's better to give users a choice but if you think it's a good default, then np => can be bundled inside the main lib and instantiated + passed in [here](https://github.com/nearform/graphql-hooks/blob/8451502568468629b038ba054727a7d037dbd962/packages/graphql-hooks/src/GraphQLClient.js#L61) 👍 

My main concern was that the lib size would no longer be that small which seems to be one of the highlights of this lib (looking at readme)

Reply:
I don't have full context on this library, but I tend to side with vaclav on this one.